CMakeLists.txt

```diff
@@ -136,11 +136,11 @@ else()
   target_include_directories(OrbitAsm INTERFACE OrbitAsm/)
 
   add_subdirectory(OrbitLinuxTracing)
+  add_subdirectory(OrbitService)
```

Comment:
Or do we need to build it on Windows?

Reply:
Yes please. It's not working yet but the plan is to run the UI without admin rights and then spawn OrbitService with elevated permissions. Especially in our remote-only use-case it's important not to need admin rights to run the UI.

Reply:
But what's the point of building OrbitService on Windows if then OrbitLinuxTracing is not built on Windows (there's a ton of Linux-specific code there), and the parts that call OrbitLinuxTracing are `#ifdef`d?

Reply:
(For context, I was moving OrbitService to Linux only because `sigaction` is not available there.)

Reply:
And why do we even need admin rights for the UI on Windows right now?

Reply:
We can also move it back later.

Reply:
But is the plan to be able to build the service with the Visual Studio compiler?

Reply:
It's the plan to make no difference between remote and local profiling on all platforms.
So yes at some point we need "a" service on Windows. But I think it's still unclear on which
level the OS-differentiation will be done. We could have a OrbitServiceWin32 target or we could have a main_win32.cpp in OrbitService, or we could have ifdefs in the existing main.cpp.